internal/core/application/types.go

```diff
@@ -188,6 +189,19 @@ func (u Unspents) ToUtxos() []explorer.Utxo {
 	return l
 }
 
+type Page domain.Page
```

Comment:
Looks like you wanted to remove domain structs from interface level, what was the motivation?

Reply:
the interface level should depend only by application in a direct way. Also, this "overridden" type is helpful to convert a page from interface level to the domain type in a clear way.